webofneeds/won-bot/src/main/java/won/bot/framework/eventbot/action/impl/mail/send/WonMimeMessageGenerator.java

```diff
@@ -18,16 +21,23 @@
 import java.io.StringWriter;
 import java.io.UnsupportedEncodingException;
 import java.net.URI;
-import java.util.List;
+import java.util.*;
 
 /**
  * This Class is used to generate all Mails that are going to be sent via the Mail2WonBot
  */
 public class WonMimeMessageGenerator {
+    private static final Logger logger = LoggerFactory.getLogger(WonMimeMessageGenerator.class);
 
     @Autowired
     private VelocityEngine velocityEngine;
 
+    private int MAX_CONVERSATION_DEPTH = 3;
+    private char QUOTE_CHAR = '>';
+
+    private static final String YOUR_MESSAGE = "You said: ";
```

Comment:
i think this can be left alone for now, i do not think we need to transfer those vars to the properties or the templates since we will have to think about integrating i18n at some point and thus will have to change the way we include "static" text anyway